LICENSE

```diff
@@ -235,3 +235,12 @@ The following components are provided under the MIT license. See project link fo
 The text of each license is also included at licenses/LICENSE-[project].txt.
 
     logrus 1.7.0: https://github.com/sirupsen/logrus MIT
+
+========================================================================
+BSD licenses
+========================================================================
+
+The following components are provided under a BSD license. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+
+   third-party/template files from golang/go: https://github.com/golang/go/tree/master/src/text/template  BSD
```

Comment:
This should be also included in `dist/LICENSE` file, please

Reply:
OK. Do I also need add a file `LICENSE-golang.txt` ?

Reply:
> OK. Do I also need add a file `LICENSE-golang.txt` ?

Yes, can you please also check that whether the exact license is `BSD-3-Clause`? BSD license has several variations